doc/psyclone_script.rst

```diff
@@ -0,0 +1,257 @@
+.. _psyclone_script:
+
+psyclone script
```

Comment:
Should at least "script" be capitalised to be consistent with other headers? Capitalising headers is a bit inconsistent throughout the section so it would be good to choose one convention and follow it.

Reply:
Done.